src/ResultsActivity/elements/PayOtherForm/PayOtherForm.js

```diff
@@ -10,47 +9,56 @@ import TextArea from '@govuk-react/text-area'
 import Button from '@govuk-react/button'
 import Fieldset from '@govuk-react/fieldset'
 import ErrorSummary from '@govuk-react/error-summary'
-import { spacing } from '@govuk-react/lib'
 
-import ButtonCancel from '../ButtonCancel'
+import { ButtonContainer, ButtonCancel } from '../../../Components/Buttons'
 import RadioGroup from '../../../Components/RadioGroup'
 import { properCaseName } from '../../../utils'
 import { FieldWithError, WhenFieldChanges, onHandleErrorClick } from '../../../final-form-govuk-helpers'
-
-const ButtonContainer = styled.div`
-  button {
-    ${spacing.responsiveMargin({ size: 3, direction: 'right' })};
-  }
-`
+import IEPCreated from '../../../IEPCreated'
 
 const commentOrCaseNote = value => {
   if (value === 'UnacceptableAbsence' || value === 'Refused') return 'case note'
   return 'comments'
 }
 
-const validateThenSubmit = submitHandler => values => {
-  const formErrors = []
+export function PayOtherForm({ user, offender, updateOffenderAttendance, absentReasons, showModal, activityName }) {
+  const { offenderNo, bookingId, eventId, eventLocationId, attendanceInfo } = offender
+  const { id, absentReason, comments } = attendanceInfo || {}
 
-  if (!values.pay) {
-    formErrors.push({ targetName: 'pay', text: 'Select a pay option' })
-  }
+  const validateThenSubmit = submitHandler => async values => {
+    const formErrors = []
+    const shouldCreateIEP = values.absentReason === 'Refused' || values.absentReason === 'UnacceptableAbsence'
```

Comment:
We should hard code this check. The absent reasons data contains a list of reasons that trigger IEPS